src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -130,10 +130,12 @@ private async Task<FormulaValue> TryHandleSet(CallNode node, SymbolContext conte
             return CommonErrors.UnreachableCodeError(node.IRContext);
         }
 
-        public override async ValueTask<FormulaValue> Visit(CallNode node, SymbolContext context)
+        public override async ValueTask<FormulaValue> Visit(CallNode node, (SymbolContext, StackMarker) context)
         {
             CheckCancel();
-            
+
+            context.Item2 = context.Item2.Inc();
```

Comment:
Item1 and Item2 are not very meaningful. It might be a good idea to make a new class, EvalVisitorContext, which contains these 2 sub-contexts as properties.

Reply:
Fixed